setup/www/resources/config/nodejs.org

```diff
@@ -45,6 +45,8 @@ server {
     }
 
     location / {
+        try_files $uri $uri/ @english_fallback;
```

Comment:
Maybe we could do `try_files $uri $uri/ @english_fallback @localized_404;`?

Where `@localized_404` is just `return 404;` without `rewrite`?